include/libopencm3/stm32/l4/rcc.h

```diff
@@ -450,7 +450,9 @@ Twelve frequency ranges are available: 100 kHz, 200 kHz, 400 kHz, 800 kHz,
 #define RCC_APB1ENR1_OPAMPEN			(1 << 30)
 #define RCC_APB1ENR1_DAC1EN			(1 << 29)
 #define RCC_APB1ENR1_PWREN			(1 << 28)
+#define RCC_APB1ENR1_CAN2_USBFSEN		(1 << 26)
```

Comment:
Given that these are not available on the same parts, I'd rather see two defines,
RCC_APB1ENR1_CAN2EN for bit 26 and _also_ RCC_APB1ENR1_USBFSEN for bit 26.